app/models/poll.rb

```diff
@@ -83,6 +83,12 @@ def initialize(poll, id, title, votes_count)
     end
   end
 
+  def reset_votes!
+    self.cached_tallies = options.map { 0 }
```

Comment:
This happens before options is updated, so this makes `[0, 0]` and not `[0, 0, 0]` in the test

Reply:
Yeah I know, I changed the tests to catch that but didn't change the implementation yet.